### DISCUSSION TITLE
How to have effectively two blogs in one site

### DISCUSSION THREAD

**Discussion by federicomenaquintero** [getpelican/pelican#3098] [Q&A]
My main blog is (say) at https://example.com.  Within it, I use a pelican category for what is effectively a second blog, and which by default gets published at https://example.com/category/second.html

However, I would like the main page of example.com to have only the main blog's entries, not the second one's.  That is, I would like to either:

* exclude the `second` category from the main page
* or explicitly include only the default category in the main page.

I found the `feed-filter` plugin to exclude a category from a feed, and it works well to exclude `second` from the main feed, but obviously I would like the HTML to be the same :)

My `index.html` template has this bit:

```
{% for article in articles_page.object_list %}
  <li>
    {% include 'article-content.html' %}
  </li>
{% endfor %}
```

It is based on the simple theme, which I tweaked a bit.

My understanding is that `articles_page` there uses whatever Pelican thinks should be in the main page, i.e. all articles in all categories; please tell me if I'm wrong.

And for example, my `category.html` template is just this:

```
{% extends "index.html" %}
{% block content_title %}
<h2>Articles in the {{ category }} category</h2>
{% endblock %}
```

So as far as I can tell, Pelican uses that, but puts only the current category's articles in `articles_page`.  If I end up modifying `index.html` to have something else to only have the default category, how should I modify `category.html` to keep it doing the right thing?

Thanks!

(In the very worst case, I can probably have two configurations for Pelican - one for the main blog, one for the second one, and then have a script to run Pelican twice, and then shuffle some files around when generating the whole thing.  But I'd like to avoid doing that if it's possible to do it from a single Pelican configuration.)

**Comment by cjhabs** [comment]
I came across something just yesterday that may help you. It was not in the main documentation but was mentioned in the FAQ and works great.
You can add a "template:" directive in your front matter to use a different template. Then you can copy an existing template and modify it without affecting the original.
template: newblog
will use template file newblog.html
